packages/tyxml-jsx/tyxml-jsx.4.5.0/opam

```diff
@@ -0,0 +1,50 @@
+opam-version: "2.0"
+synopsis: "JSX syntax to write TyXML documents"
+description: """
+```reason
+open Tyxml;
+let to_reason = <a href="reasonml.github.io/"> "Reason!" </a>
+```
+ 
+The TyXML JSX allow to write TyXML documents with reason's JSX syntax. 
+It works with textual trees, virtual DOM trees, or any TyXML module.
+"""
+maintainer: ["dev@ocsigen.org"]
+authors: ["The ocsigen team"]
+license: "LGPL-2.1 with OCaml linking exception"
+homepage: "https://github.com/ocsigen/tyxml"
+doc: "https://ocsigen.org/tyxml/latest/manual/intro"
+bug-reports: "https://github.com/ocsigen/tyxml/issues"
+depends: [
+  "dune" {>= "2.0"}
+  "ocaml" {>= "4.04"}
+  "tyxml" {version}
+  "tyxml-syntax" {version}
+  "alcotest" {with-test}
+  "reason" {with-test}
+  "ppxlib"
+]
+build: [
+  ["dune" "subst"] {pinned}
```

Comment:
```suggestion
  ["dune" "subst"] {dev}
```

Reply:
cc @rgrinberg , this is generated by dune.

Reply:
yes, this as been fixed in dune 2.7. opam file generation in dune just requires you to upgrade to `(lang dune 2.7)` to generated the fixed version